docs/libcurl/symbols-in-versions

```diff
@@ -886,6 +888,7 @@ CURLOPT_WS_OPTIONS              7.86.0
 CURLOPT_XFERINFODATA            7.32.0
 CURLOPT_XFERINFOFUNCTION        7.32.0
 CURLOPT_XOAUTH2_BEARER          7.33.0
+CURLOPTDEPRECATED               7.89.0
```

Comment:
7.87.0 and merge asap?

Reply:
Yes, done and rebased.
Maybe wait for the CI tests to complete before merge?

Reply:
:+1:

Reply:
CI tests complete now. Failed tests seems not related.